solutions/javascript/init/code/app/main.js

```diff
@@ -0,0 +1,7 @@
+const net = require("net");
+
+const server = net.createServer((socket) => {
+  socket.pipe(socket);
+});
+
+server.listen(6379, "127.0.0.1");
```

Comment:
(This is the main file of interest `main.js` for stage 1).

Reply:
Is there meant to be comments explaining what is occurring here or is the user expected to know that this is just setting up a TCP server that echos the messages back to the client?

Reply:
Hey @ijjk - good catch re: the echo-ing back functionality, I'll remove that. 

The TCP server part should be clear to users since this code is presented in the context of the "bind to a port" stage instructions: 

<img width="659" alt="Screenshot 2022-07-27 at 13 47 08" src="https://user-images.githubusercontent.com/3893573/181198710-31afbf02-f513-4f7c-9c66-54b1d93b3642.png">

<img width="938" alt="Screenshot 2022-07-27 at 13 46 53" src="https://user-images.githubusercontent.com/3893573/181198742-914c6d6e-afd3-4405-99f0-77c6417b38fc.png">

Reply:
@ijjk okay, the `socket.pipe` call should be removed now! I also renamed the `socket` parameter to `connection` since that's closer to the official node docs.

Here's the essence of what was changed: 

<img width="1154" alt="Screenshot 2022-07-27 at 14 17 18" src="https://user-images.githubusercontent.com/3893573/181204328-3824c842-a485-4ac5-9d8c-fdf2d8015eb1.png">